integrations/issue_test.go

```diff
@@ -116,17 +116,70 @@ func testNewIssue(t *testing.T, session *TestSession, user, repo, title string)
 	link, exists := htmlDoc.doc.Find("form.ui.form").Attr("action")
 	assert.True(t, exists, "The template has changed")
 	req = NewRequestWithValues(t, "POST", link, map[string]string{
-		"_csrf": htmlDoc.GetCSRF(),
-		"title": title,
+		"_csrf":   htmlDoc.GetCSRF(),
+		"title":   title,
+		"content": content,
+	})
+	resp = session.MakeRequest(t, req, http.StatusFound)
+
+	issueURL := RedirectURL(t, resp)
+	req = NewRequest(t, "GET", issueURL)
+	resp = session.MakeRequest(t, req, http.StatusOK)
+
+	htmlDoc = NewHTMLParser(t, resp.Body)
+	val := htmlDoc.doc.Find("#issue-title").Text()
+	assert.Equal(t, title, val)
+	val = htmlDoc.doc.Find(".comment-list .comments .comment .render-content p").First().Text()
+	assert.Equal(t, content, val)
+
+	return issueURL
+}
+
+func testIssueAddComment(t *testing.T, session *TestSession, issueURL, content, status string) {
```

Comment:
nit: it's not obvious what values are valid for `status` (without looking through other parts of the codebase), and if I see something like `
testIssueAddComment(t, sess, url, "hello", "close")` in a test, it's not obvious that this will close the issue, in addition to making a comment.

Could we instead do:
```
func testIssueAddComment(t, session, issueURL, content) {
	testIssueAndCommentWithStatus(t,session,issueURL,content,"")
}

func testIssueAddClosingComment(t, session, issueURL, content) {
	testIssueAndCommentWithStatus(t,session,issueURL,content,"close")
}

// helper function
func testIssueAddCommentWithStatus(t, session, issueURL, content, status) {
	...
}
```

I think this will make it easier for future tests to reuse these helpers, and will make tests more readable.